wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Linker.kt

```diff
@@ -449,6 +449,10 @@ class Linker {
     val error = StringBuilder()
     error.append(message)
 
+    val contextStack = this.contextStack.toMutableList()
+    if (contextStack.size > 1) {
+      contextStack.removeAll { it is ProtoFile }
```

Comment:
perfect

Reply:
<img width="535" alt="Screen Shot 2020-07-17 at 21 52 56" src="https://user-images.githubusercontent.com/133019/87842103-e4973680-c877-11ea-97a3-475e30a0bfb8.png">

I spoke too soon! Needs to check if any is not a ProtoFile.
```
    if (contextStack.any { it !is ProtoType }) {
      contextStack.removeAll { it is ProtoFile }
    }
```

Reply:
Fixed, thanks for checking!